nautobot/project-static/bootstrap-3.4.1-dist/css/bootstrap.css

```diff
@@ -1522,29 +1522,29 @@ blockquote.pull-right .small:after {
   content: "\00A0 \2014";
 }
 address {
-  margin-bottom: 20px;
+  margin-bottom: 24px;
   font-style: normal;
-  line-height: 1.42857143;
+  line-height: 1.71428572;
 }
 code,
 kbd,
 pre,
 samp {
-  font-family: Menlo, Monaco, Consolas, "Courier New", monospace;
+  font-family: "Ubuntu Mono", monospace;
```

Comment:
Do we need/want to bundle `Ubuntu Mono` font as well?

Reply:
I dont know if we are using monospace fonts anywhere, but I think we def should.

Reply:
We definitely are - any JSON fields, rendered config-contexts, etc.

Reply:
Good point, @glennmatthews, `Ubuntu Mono` should be served from static files just as `Ubuntu` is. That's my oversight